strongbox-testing/strongbox-testing-core/src/test/java/org/carlspring/strongbox/artifact/generator/ArtifactDeployerTest.java

```diff
@@ -26,6 +27,7 @@
  */
 @RunWith(SpringJUnit4ClassRunner.class)
 @ContextConfiguration(classes = TestingCoreConfig.class)
+@Ignore
```

Comment:
What is the reason for this test to be ignored? Is it currently failing? I assume this is related to the fact that there is no running Jetty?